athena_cfn/README.md

```diff
@@ -0,0 +1,190 @@
+# Query Quilt Package Metadata in AWS Athena
+
+The code in this example shows you how to enable use of the "Queries"
+tab of the Quilt Catalog for querying Quilt [metadata](https://docs.quiltdata.com/catalog/metadata) for a given bucket using [Athena SQL](https://docs.quiltdata.com/advanced/athena).
+
+### Validating CloudFormation files
```

Comment:
This section, for readability, should probably be bumped much later, when the customer might actually generate the templates. Also we should remind customers to switch to their desired environment before `pip install` etc.

Reply:
Moved